### DISCUSSION TITLE
Z-Wave Kit HA Serial Connection not working

### DISCUSSION THREAD

**Discussion by adamazing-dev** [tube0013/tube_gateways#211] [Q&A] [answered]
Hi,

I just received my ZW Kit and installed the ZB7Pro on it. Powering and Connecting via RJ45 worked and I can open http://IP which shows me the serial connection status and gives me a reset button besides the log. When I try to connect via tcp://IP:6638 or socket://IP.6638 on my HA I got connection failed while the log of ESP shows me connected and disconnected. I reflashed the ZW-Kit Binary, but same issue.

<img width="1118" alt="tubeszw1" src="https://github.com/user-attachments/assets/ab915397-e1f8-45ab-8918-dae94cf8cb3a">
<img width="387" alt="tubeszw2" src="https://github.com/user-attachments/assets/22c79844-810c-4140-b8b9-c6acbffe7759">
<img width="392" alt="tubeszw3" src="https://github.com/user-attachments/assets/85d5c50a-de09-4a7a-8e57-d9a454a2111a">

**Comment by tube0013** [comment, marked as answer]
You must use the Z-Wave JS UI addon or other type of install to be able to use a tcp:// device. Unfortunately the built in Z-Wave JS in HA only support directly connected usb/serial devices. 

Also ignore any discovered Zigbee Home Automation devices. There is some old discovery code in HA that recognizes any TubesZB device as a zigbee coordinator.

**Comment by adamazing-dev** [comment]
Thanks for the fast reply. This resolved the issue. It's working using the addon :-)
